.jshintrc

```diff
@@ -12,6 +12,7 @@
   "undef": true, // Require all non-global variables be declared before they are used.
   "unused": false, // Warn unused variables.
   "strict": true, // Require `use strict` pragma in every file.
+  "quotmark": "single", // Require only single quotes
```

Comment:
@jloveland I don't think it's really that necessary to just make a temporary commit..I'd rather wait a few days for ESLint.